### DISCUSSION TITLE
Printing HTML documents using columns vs n-col layout

### DISCUSSION THREAD

**Discussion by rressler** [quarto-dev/quarto-cli#7743] [Q&A]
### Description

I am putting multiple plots or images in a row, rendering to HTML, and then printing the document.

With  #| layout-ncol: 2, the plots show up fine in Chrome. However when printing, they lose the 2 column layout and appear one after the other.

With the column divs (below), they also show up fine when viewed in Chrome and when printed they retain their 2 column layout shown in the browser.

Is this the expected behavior for #| layout-ncol: 2?
Thanks,
Richard

::: columns
::: {.column width="50%"}
```{r}

```
:::

::: {.column width="50%"}
```{r}

```
:::
:::

**Comment by cscheid** [comment]
CSS makes a distinction between screen layouts and print layouts (and our CSS is not optimized for printing). If you need the layout to be preserved, I think your best bet is to use PDF formats.

**Comment by rressler** [comment]
Thanks!

From: Carlos Scheidegger ***@***.***>
Date: Wednesday, November 29, 2023 at 9:14 AM
To: quarto-dev/quarto-cli ***@***.***>
Cc: Richard Ressler ***@***.***>, Author ***@***.***>
Subject: Re: [quarto-dev/quarto-cli] Printing HTML documents using columns vs n-col layout (Discussion #7743)

External Email: Use caution with links and attachments.

CSS makes a distinction between screen layouts and print layouts (and our CSS is not optimized for printing). If you need the layout to be preserved, I think your best bet is to use PDF formats.

**Reply by mcanouil** [reply]
@rressler for your information, when you reply from email, do not include previous content as it gets added to your message on GitHub.
